crates/omnix-cli/src/command/show.rs

```diff
@@ -1,11 +1,120 @@
+use std::collections::BTreeMap;
+
+use anyhow::Context;
 use clap::Parser;
+use colored::Colorize;
+use nix_rs::{
+    command::NixCmd,
+    flake::{outputs::Val, url::FlakeUrl, Flake},
+};
+use tabled::{
+    settings::{
+        style::{HorizontalLine, VerticalLine},
+        Style,
+    },
+    Table, Tabled,
+};
 
 #[derive(Parser, Debug)]
-pub struct ShowConfig {}
+pub struct ShowConfig {
+    /// The flake url to show outputs for
+    pub flake_url: FlakeUrl,
```

Comment:
```suggestion
    /// The flake to show outputs for
    #[arg(name = "FLAKE")]
    pub flake_url: FlakeUrl,
```

cf. https://github.com/juspay/omnix/blob/a5d2c85fcbdf8ceab561ff4128934d60636b12d5/crates/omnix-cli/src/command/health.rs#L7-L9

(It is not always going to be URL, as you can use directory paths as well)